src/transformers/default-schema-transformer.ts

```diff
@@ -276,3 +279,15 @@ function handleQuestionsWithObsComments(sectionQuestions: Array<FormField>): Arr
 
   return augmentedQuestions;
 }
+
+function handleDiagnosis(question: FormField) {
+  if ('dataSource' in question.questionOptions && question.questionOptions['dataSource'] === 'diagnoses') {
+    question.questionOptions.datasource = {
+      name: 'problem_datasource',
+      config: {
+        class: question.questionOptions.diagnosis.conceptClasses,
+      },
+    };
+    delete question.questionOptions['dataSource'];
+  }
```

Comment:
Are form designers expected to provide the data source configuration as a requirement? Can we not handle cases where none is provided?

Reply:
Let's provide a default data source config in cases where none was provided.